README.md

```diff
@@ -314,15 +330,15 @@ In this case the Kafka Cluster service is `my-cluster-kafka-bootstrap:9092`.
 
 Use this information to create the Kogito Data Index resource. 
 
-#### Install Data Index with Kogito CLI
+#### Install Data index Service with Kogito CLI
```

Comment:
I'd rather leave the `index` with capital **I** to follow the "convention" from the Wiki: https://github.com/kiegroup/kogito-runtimes/wiki/Data-Index-Service